testcases/OpTestEEH.py

```diff
@@ -36,140 +36,19 @@
 import sys
 import os
 
-from common.OpTestBMC import OpTestBMC
-from common.OpTestIPMI import OpTestIPMI
-from common.OpTestConstants import OpTestConstants as BMC_CONST
-from common.OpTestError import OpTestError
```

Comment:
Can we have this OpTestError imported, in-order to properly raise the exceptions in this EEH tests.